app/src/main/java/com/github/se/cyrcle/model/parking/ParkingViewModel.kt

```diff
@@ -34,44 +42,51 @@ class ParkingViewModel(
   fun fetchImageUrl(path: String): StateFlow<String?> {
     val imageUrlFlow = MutableStateFlow<String?>(null)
     viewModelScope.launch {
-      try {
-        imageUrlFlow.value = imageRepository.getUrl(path)
-      } catch (e: Exception) {
-        imageUrlFlow.value = null
-      }
+      imageUrlFlow.value =
+          try {
+            imageRepository.getUrl(path)
+          } catch (e: Exception) {
+            null
```

Comment:
Might be worth the effort of error handling? I'm not extremely familiar with this class so I'm not sure though